onnxscript/ir/passes/common/shape_inference_test.py

```diff
@@ -62,30 +64,30 @@ def test_pass_with_initializers(self):
             ),
         ]
 
+        tape = ir.tape.Tape()
+
         # Shape and type are not explicitly set for the initializer but it should still work
         initializer = ir.Value(
             name="initializer", const_value=ir.tensor([[2, 3]], dtype=ir.DataType.FLOAT)
         )
-
-        add_node = ir.Node("", "Add", inputs=[*inputs])
-        mul_node = ir.Node("", "Mul", inputs=[add_node.outputs[0], initializer])
+        val_add = tape.op("Add", inputs=inputs)
+        val_mul = tape.op("Mul", inputs=[val_add, initializer])
```

Comment:
could we add a test demonstrating usage of tape.op_multi_output() ?

Reply:
Added a test